src/docfx/lib/JsonUtility.cs

```diff
@@ -112,7 +115,7 @@ public static (List<Error>, object) ToObject(JToken token, Type type)
                 var serializer = new JsonSerializer
                 {
                     NullValueHandling = NullValueHandling.Ignore,
-                    ContractResolver = DefaultDeserializer.ContractResolver,
+                    ContractResolver = new JsonContractResolver(transform),
```

Comment:
>JsonContractResolver [](start = 43, length = 20)

will this cause perf problems? use `t_transform` instead?

Reply:
Yes, it would, will update it

---
In reply to: [210817828](https://github.com/dotnet/docfx/pull/3209#discussion_r210817828) [](ancestors = 210817828)